.dockerignore

```diff
@@ -12,3 +12,4 @@ venv/
 /.github/
 /netlify.toml
 /setup/
+/tests/
```

Comment:
Why? The size they occupy is really negligible.

Reply:
I both agree and disagree with you on that. It is because redash's docker image is too large, however, I will rebase it because .circleci/config became to need unnecessary change. 🤔